LDAR_Sim/src/initialization/input_manager.py

```diff
@@ -185,21 +164,28 @@ def parse_parameters(self, new_parameters_list):
                     if len(new_parameters['programs']) > 0:
                         programs = programs + new_parameters.pop('programs')
 
-                check_types(default_global_parameters, new_parameters, omit_keys=['programs'])
+                check_types(self.simulation_parameters, new_parameters, omit_keys=['programs'])
                 self.simulation_parameters.update(new_parameters)
 
             elif new_parameters['parameter_level'] == 'program':
+                if 'default_parameters' not in new_parameters:
+                    def_file = 'p_default.yml'
+                else:
+                    def_file = new_parameters['default_parameters']
```

Comment:
I don't think it would be our job to manage the user curated defaults, we can manage the defaults and the documentation for additional releases , and it would be expected that the user updates their parameters accordingly.  I fully agree, that most users should use the defaults, the ability to add your own defaults comes out of a need that we have seen with Highwood, where we may add different parameters to sensors or deployment methods that are not in the native V2 of LDAR-Sim. We would still want to typecheck these additional params, but also allow changing via the input yaml file. 

Highwood (and our customers) would actually love if you could use "plugin" type modules with the native LDAR Sim V2 instead of having forks, this kind of moves towards this.